### PR TITLE
Story#538

### PR DESCRIPTION
**Pull request by rejneesh1**

### CODE REVIEW

File: src/components/evidence-all-list/evidence-all-list.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  </div>
+
+  <div [ngSwitch]="selectedTab">
+    <div *ngSwitchCase="'evidence'">
+      <!-- //for images -->
+      <div>
+        <ion-grid>
+          <ion-row>
+            <ion-col col-12>
+              <h5>Images</h5>
```
Comment (kiranharidas187) [inline src/components/evidence-all-list/evidence-all-list.html]: use translate

File: src/components/attachments/attachments.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!-- Generated template for the AttachmentsComponent component -->
+
+<ion-card class="thumbnail-card" [ngSwitch]="true">
+  <img
+    [src]="url"
+    *ngSwitchCase="extension==='jpg'
```
Comment (kiranharidas187) [inline src/components/attachments/attachments.html]: check for all formats of image. Refer imageListingcomponent.

File: src/components/attachments/attachments.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    ||extension==='png'
+    ||extension==='jpeg'"
+    (click)="openImage(url)"
+  />
+  <fa
```
Comment (kiranharidas187) [inline src/components/attachments/attachments.html:9]: check for all formats of image. Refer imageListingcomponent.

File: src/components/attachments/attachments.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    name="file-video-o"
+    class="attachmentIcon"
+    *ngSwitchCase="extension==='mp4'
+    ||extension==='m4v'
+    ||extension==='avi'
+    ||extension==='flv'"
+    (click)="playVideo(url)"
+  ></fa>
+
+  <fa
```
Comment (kiranharidas187) [inline src/components/attachments/attachments.html:16]: check for all formats of image. Refer imageListingcomponent.

File: src/components/evidence-all-list/evidence-all-list.ts [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+  }
+
+  getAllEvidence() {
+    let url = AppConfigs.observationReports.allEvidence;
+
+    this.apiService.httpPost(
+      url,
+      this.payload,
+      (success) => {
+        this.utils.stopLoader();
```
Comment (kiranharidas187) [inline src/components/evidence-all-list/evidence-all-list.ts:70]: Where is the start loader??

File: src/components/attachments/attachments.html (hunk 2 above)
Comment (rejneesh1) [reply src/components/attachments/attachments.html]: for image only three images extension is there jpg,png,jpeg
for video and audio added all other extension

File: src/components/evidence-all-list/evidence-all-list.ts (hunk 5 above)
Comment (rejneesh1) [reply src/components/evidence-all-list/evidence-all-list.ts:70]: added

**Review by kiranharidas187 — APPROVED** [review]

**Pull request merged**